database/migrations/2017_06_14_151728_create_order_products_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOrderProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_products', function (Blueprint $table) {
```

Comment:
Need add user_id for this table. user don't need relationship with user, if guest order, we store user_id is null